bin/run.sh

```diff
@@ -72,8 +76,17 @@ cp "${manifest_file}" "${manifest_file_bak}"
 cp "${gleam_file}" "${gleam_file_bak}"
 rm -fr "$solution_dir"/build
 cp -r "$root_dir"/packages/build "$solution_dir"/build
-cp "$root_dir"/packages/manifest.toml "${manifest_file}"
-cat "$root_dir"/packages/gleam.toml | sed "s/name = \".*\"/name = \"$underscore_slug\"/" > "${gleam_file}"
+if $local; then
```

Comment:
This runs the `true` or `false` command, which requires a fork-exec. Another common approach is C-style flags. You can set `local` to 0 or 1 then test `if (( local )); then`

```suggestion
if [[ $local == true ]]; then
```